src-vue/src/views/DeveloperView.vue

```diff
@@ -29,6 +34,28 @@ export default defineComponent({
                 type: 'error',
                 position: 'bottom-right'
             });
+        },
+        async toggleReleaseCandidate() {
+            // Flip between RELEASE and RELEASE_CANDIDATE
+            if (this.$store.state.release_canal === ReleaseCanal.RELEASE) {
+                this.$store.state.release_canal = ReleaseCanal.RELEASE_CANDIDATE;
+            }
+            else {
+                this.$store.state.release_canal = ReleaseCanal.RELEASE;
+            }
+
+            // Update current state so that update check etc can be performed
+            this.$store.commit("initialize");
```

Comment:
Shouldn't you be calling `checkNorthstarUpdates` instead of `initialize`?
`initialize` should be only called on application boot.

Reply:
Fixed in 3398c073eb57c4309770a82b1193c198744d7ad6